pyproject.toml

```diff
@@ -56,6 +56,7 @@ allenmouse = ["allensdk"]
 allenmouse_barrels = [
     "allensdk",
     "voxcell"]
+perens_stereotaxic_mri_mouse = ["py7zr"]
```

Comment:
hopefully we don't need an extra dependency here, if we move to `pooch`?

Reply:
I think we still do. I will move the script to pooch but this file is in 7zip format which I think requires the py7zr dependency to decompress.

Reply:
You're right - yep, thank yoU!